in_toto/log.py

```diff
@@ -20,22 +20,26 @@
 
 logging.basicConfig(level=in_toto.settings.LOG_LEVEL, format='%(message)s')
 
+def debug(msg):
+  """ Verbose debugging """
+  logging.debug("DEBUG: {}".format(in_toto.util.color_code(msg, 10)))
+
 def info(msg):
   """Verbose user feedback. """
-  logging.info("{}".format(msg))
+  logging.info("INFO: {}".format(in_toto.util.color_code(msg, 20)))
 
 def warn(msg):
   """Verbose user warning. """
-  logging.warn("WARNING: {}".format(msg))
+  logging.warn("WARNING: {}".format(in_toto.util.color_code(msg, 30)))
 
 def error(msg):
   """Prints unexpected errors """
-  logging.error("ERROR: {}".format(msg))
+  logging.error("ERROR: {}".format(in_toto.util.color_code(msg, 40)))
 
 def pass_verification(msg):
   """Prints passing verification routines. """
-  logging.critical("PASSING: {}".format(msg))
+  logging.critical("PASSING: {}".format(in_toto.util.color_code(msg, 50)))
```

Comment:
I think `pass_verification` should have a more positively connotated color. 
Also, could you pass the constant names instead of the hardcoded values as `log.level` parameter? Or maybe it makes even more sense to just pass the color, defined in a constant, e.g. `log.GREEN`, ...

Reply:
Yup, changed the message for `pass_verification` to be GREEN instead of RED.